charts/pg-db/templates/cluster.yaml

```diff
@@ -368,6 +368,7 @@ spec:
     image: {{ .Values.pmm.image.repository }}:{{ .Values.pmm.image.tag }}
     serverHost: {{ .Values.pmm.serverHost }}
     secret: {{ include "pg-database.fullname" . }}-pmm-secret
+    querySource: {{ .Values.pmm.querySource }}
```

Comment:
From what I saw in the operator PR, changes were made in the CRD when introducing this new field, so please copy the new CRD under pg-operator/crds/

Reply:
[1e2ae7c](https://github.com/percona/percona-helm-charts/pull/368/commits/1e2ae7c1b21bcaf6848656684403b589c9dcdeb4)